analyzer_test.go

```diff
@@ -9,45 +9,55 @@ import (
 	"os"
 	"path/filepath"
 	"testing"
+
+	"github.com/Masterminds/semver"
+	"github.com/golang/dep/test"
 )
 
-func TestDeriveManifestAndLock(t *testing.T) {
-	dir, err := ioutil.TempDir("", "dep")
+func TestAnalyzerInfo(t *testing.T) {
```

Comment:
I tend to agree with @freeformz, though this also falls under the heading of "write code that's easy to delete," so i'm not too bothered.